packages/cardano-services/test/ChainHistory/DbSyncChainHistoryProvider/__snapshots__/ChainHistoryBuilder.test.ts.snap

```diff
@@ -99,19 +99,19 @@ Map {
 
 exports[`ChainHistoryBuilder queryProtocolParams query protocol parameters from last epoch 1`] = `
 Object {
-  "coinsPerUtxoByte": NaN,
```

Comment:
This happened because I replaced the type from number to bigint, number return NaN when undefined, however, bigint doesn't have NaN so I am defaulting to 0 when the field is missing. This in fact could be a problem I am looking into this.

Reply:
ok addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/388#discussion_r951056969